packages/components/src/components/modal/modal.scss

```diff
@@ -1,19 +1,15 @@
 /** @define dc-modal; weak */
 .dc-modal {
     @include mobile {
-        position: absolute;
-        top: 0;
-
-        .dc-modal__container {
-            min-width: unset;
-            top: 2rem;
-        }
+        width: calc(100vw - 4.8em);
     }
 
     &__container {
         display: flex;
         flex-direction: column;
-        min-width: 400px;
+        @include desktop {
```

Comment:
Can u move the desktop `mixin` wrapper to L21?

Reply:
Absolutely.